From 72ba0adb279974e6677e38f4cf71f371a147e240 Mon Sep 17 00:00:00 2001
From: "pengzhao.liu" <pengzhao.liu@amlogic.com>
Date: Tue, 20 Jun 2023 14:00:07 +0800
Subject: [PATCH] kernel: Remove Makefile autoreview script [2/2]

PD#SWPL-128588

Problem:
Cannot copy files when compiling

Solution:
Remove Makefile autoreview script

Verify:
no

Change-Id: I40350c71e41a392167f7e85dddcbc47f395fd004
Signed-off-by: pengzhao.liu <pengzhao.liu@amlogic.com>
---
 Makefile | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/Makefile b/Makefile
index 46bf5fae57e1..3897c4d55113 100644
--- a/Makefile
+++ b/Makefile
@@ -2113,23 +2113,8 @@ endif # config-build
 endif # mixed-build
 endif # need-sub-make
 
-ifdef CONFIG_AMLOGIC_DRIVER
 PHONY += FORCE
 FORCE:
-	$(Q)-cp $(srctree)/scripts/amlogic/pre-commit $(srctree)/.git/hooks/pre-commit
-	$(Q)-cp $(srctree)/scripts/amlogic/pre-commit-common_drivers $(srctree)/$(COMMON_DRIVERS_DIR)/.git/hooks/pre-commit
-	$(Q)-chmod +x $(srctree)/$(COMMON_DRIVERS_DIR)/.git/hooks/pre-commit
-	$(Q)-mkdir -p $(srctree)/$(COMMON_DRIVERS_DIR)/scripts/amlogic
-	$(Q)-cp $(srctree)/scripts/amlogic/licence_pre.pl $(srctree)/$(COMMON_DRIVERS_DIR)/scripts/amlogic
-	$(Q)-cp $(srctree)/scripts/amlogic/licence_check.pl $(srctree)/$(COMMON_DRIVERS_DIR)/scripts/amlogic
-	$(Q)-cp $(srctree)/scripts/amlogic/merge_pre_check.pl $(srctree)/$(COMMON_DRIVERS_DIR)/scripts/amlogic
-	$(Q)-cp $(srctree)/scripts/checkpatch.pl  $(srctree)/$(COMMON_DRIVERS_DIR)/scripts/
-	$(Q)-cp $(srctree)/scripts/spelling.txt  $(srctree)/$(COMMON_DRIVERS_DIR)/scripts/
-	$(Q)-cp $(srctree)/scripts/const_structs.checkpatch  $(srctree)/$(COMMON_DRIVERS_DIR)/scripts/
-else
-PHONY += FORCE
-FORCE:
-endif
 
 # Declare the contents of the PHONY variable as phony.  We keep that
 # information in a variable so we can use it in if_changed and friends.
-- 
2.25.1

